sql/moz-fx-data-shared-prod/firefox_desktop/newtab/metadata.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
You should reduce the explicit metadata to just `workgroup_access` e.g. https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/telemetry_derived/events_live/metadata.yaml